addons/analytic/models/analytic_mixin.py

```diff
@@ -32,9 +37,36 @@ def init(self):
             self.env.cr.execute(query)
         super().init()
 
+    @api.model
+    def fields_get(self, allfields=None, attributes=None):
+        """ Hide analytic_distribution_search from filterable/searchable fields"""
+        res = super().fields_get(allfields, attributes)
+        if res.get('analytic_distribution_search'):
+            res['analytic_distribution_search']['searchable'] = False
+        return res
```

Comment:
@odoo/rd-framework-py 
Shouln't JSON fields not be searchable by default, except when specified explicitely?
Otherwise the ORM doesn't know what to search in there

Reply:
> Otherwise the ORM doesn't know what to search in there

It still searches on the plain text of the JSON. It is far from perfect at all but it still works in many cases (`like` like operator for example). In stable, IMO, we shouldn't change this. Maybe in master it makes sense.

Is it really important to hide this field? Any search on `analytic_distribution_search` and `analytic_precision` works fine anyway.

Reply:
> Is it really important to hide this field? Any search on `analytic_distribution_search` and `analytic_precision` works fine anyway.

It is just confusing to have the same field with a different label in the searchable fields, there are already enough fields on the models that have an analytic distribution